src/hello_imgui/hello_imgui_assets.h

```diff
@@ -47,6 +49,17 @@ AssetFileData LoadAssetFileData(const char *assetPath);
 void FreeAssetFileData(AssetFileData * assetFileData);
 // @@md
 
+// Function type to redirect asset loads. Function receives a path and 
+// returns an AssetFileData structure. By default, it points to 
+// DefaultLoadAssetFileData.
+typedef AssetFileData (*LoadAssetFileDataFunc)(const char *assetPath);
+
```

Comment:
Since I have to emit Python bindings for all other code in this library, I cannot use bare C function pointers. 

Instead I will need to be replaced by a `std::function` (which is essentially the same except that it can be bound using Python).